lib/models.js

```diff
@@ -620,7 +620,7 @@ class Cardinality {
   constructor(min, max) {
     this._min = min; // number
     this._max = max; // number|undefined
-    this._history = []; // dictionary[] (min,max,source)
+    //additional properties: source (DataElement), history (Cardinality[])
```

Comment:
Based on your commit message, you changed `source` to be an `Identifier`, right?  Or was that only for the other classes?